walkthroughs/week-3-server/step-1-servlets-walkthrough.md

```diff
@@ -131,7 +131,7 @@ Second, using a framework involves a lot of setup and configuration. Code is
 hard enough to debug, even without fighting with framework config files!
 
 Third, most of the Google Cloud tutorials use servlets. So anything you find in
-a Google Cloud tutorial should "just work" with the code you during SPS.
+a Google Cloud tutorial should "just work" with the code you write during STEP.
```

Comment:
Feel free to fix this typo over in the SPS repo as well! https://github.com/google/software-product-sprint

Reply:
Done, thanks!